TADashboard/src/main/java/page_objects/DataProfilesPage.java

```diff
@@ -0,0 +1,26 @@
+package page_objects;
+
+import com.logigear.control.common.imp.CheckBox;
+import com.logigear.control.common.imp.Link;
+import utils.common.Constants;
+
+public class DataProfilesPage {
+    private CheckBox dynamicCheckColumn = new CheckBox("xpath=//table[@class='GridView']/tbody/tr/td[text()='%s']/preceding-sibling::td/input");
```

Comment:
you define it as Checkbox but you call it Column, so what it finally is?

Reply:
Yeah OK. Thank you so much.